.mk/k8s.mk

```diff
@@ -143,10 +143,22 @@ k8s-forward:
 	@$(kubectl) port-forward $$($(kubectl) get pods | grep $(pod) | cut -d \  -f1) $(port1):$(port2)
 
 .PHONY: k8s-check
-k8s-check:
+k8s-check: k8s-icmp-check k8s-vpn-check
+
+.PHONY: k8s-icmp-check
```

Comment:
Should we also update [example doc]( https://github.com/networkservicemesh/networkservicemesh/blob/7f7a7730a5d2ea0dce04c7819647458a92c35195/docs/vagrant/guide-vagrant.md#verify-deploying-the-examples) to use `k8s-icmp-check` / `k8s-vpn-check` ?

Reply:
I didn't change the behaviour of `make k8s-check` it still runs 2 checks as it was before. Our bash-checkers also require some simplification, I intended to do it in further PRs

Reply:
in the doc for verifying `icmp example`need to execute `make k8s-check`. Script `test/shell/icmp_test.sh` uses for verifying  `make k8s-icmp-check`. I think examples from docs and examples from CI should be equal.